ROADMAP.md

```diff
@@ -69,18 +69,73 @@ There's a loose collection of some BigchainDB 2.0 "TODO" items in the GitHub org
 
 ## Some Goals for BigchainDB 2.1 and Beyond
 
-- Document how to run a production validator so that it's protected from denial-of-service (DoS) attacks and similar issues. See what Tendermint recommends (i.e. "sentry nodes", see issue https://github.com/bigchaindb/bigchaindb/issues/2333).
-- Performance optimization by removing unneeded deepcopy operations.
-- Example projects showing how to use BigchainDB with other decentralized systems, especially Ethereum.
-- Refactoring, especially of the transaction-related code.
+### Short-Term Goals - Low-Hanging Fruits
+
+These are features which are solving a very specific problem. These features, while not scoped, are expected to be small to medium in implementation complexity.
+
+#### Priority 1
+
+- Better node deployment and setup process. Analyze and ask users their issues. Define an action plan (BEP).
+- Privacy using encryption, using plugins
+  - node level encryption
+  - peer to peer encryption on same node - needs KMS
+
+#### Priority 2
+
+- Plugin architecture for transaction validation. Refactor the tx code: BEP-9. Generalize the codebase. Dynamic loading of plugins using validator election.
+- Scale-up and scale-out strategy. Define limits - run infra load tests.
```

Comment:
Defining scale-up and scale-out strategies for nodes in a network expecting larger loads. As a first step, running load tests to find infrastructure limit.